src/options_menu.c

```diff
@@ -103,34 +103,44 @@ enum
     MENU_OPTION_NO
 };
 
+enum OptionsMenuStates {
+    OPTIONS_MENU_INIT,
+    OPTIONS_MENU_MAIN,
+    OPTIONS_MENU_EXIT,
+    OPTIONS_MENU_PRE_HINT_SELECTION,
+    OPTIONS_MENU_HINT_SELECTION,
+    OPTIONS_MENU_DISPLAY_HINT,
```

Comment:
The unknown enum for 6 can be defined as something like `OPTIONS_MENU_UNKNOWN` rather than skipping it.

Reply:
Added `OPTIONS_MENU_UNKNOWN_6`